src/plugins/data/public/ui/typeahead/_index.scss

```diff
@@ -1 +1,3 @@
+$typeaheadConjunctionColor: #7800A6;
```

Comment:
You can remove this line and just set the `conjunction` color in `_suggestion.scss` to `$euiColorVis3`. 

The new EUI suggest component uses the Vis purple color as well, so it's inevitable that it ends up this way :)

Reply:
done